docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,187 @@
 # Getting started
 
-TODO
+## 1 Prerequisites
```

Comment:
I have removed the numbers, but only because of consistency with other Canonical documentation.

The navigation will only be on the right if you have a widescreen and maximize your browser. The doc site uses responsive web design, meaning if the browser is not wide enough, the table of contents simply won't show on the right. Based on my experiment, it will be gone if it's less than 1312 pixels, and for a 2560 wide screen, people usually split the screen by putting docs on the one side and then IDE/terminal on the other. In this case, if the IDE is 50% width of the screen, there is no table of contents.

I think the numbers help, especially for long articles, when you are in the middle of a section, you see another title but because it's long, you don't know if it's an H1 H2 or H3 heading, so you are kind of lost.